### DISCUSSION TITLE
Table styles width of cells in head with important markup (17.18.9)

### DISCUSSION THREAD

**Discussion by chaerror** [primefaces/community#2714] [PrimeNG]
Hi, I have upgraded my web application from PrimeNg version 11 to version 17.18.9. I have a problem with styles that are inserted directly into the head tag even with the important markup (width of cells). This overloads my own styles with an important tag as well. 

How to disable this. I haven't found a solution yet. Can someone please advise me?

Thanks for any feedback

![Image_1a](https://github.com/user-attachments/assets/e3120ea1-d344-4130-860f-0f536c2f4955)
![Image_1b](https://github.com/user-attachments/assets/f028343c-d8b2-4a72-8e9c-27612463d92d)

**Comment by qwadrox** [comment]
Hi @chaerror! You won't be able to modify these styles as they're being injected by the table component under the hood, and they're essential to make the columns resizable. I don't know what you were trying to achieve, what your styles are, but generally it is not recommended to modify or delete these injected styles.
If you can provide a minimal example with stackblitz (include your styles and requirement), I might be able to recommend you a workaround, so you can have the same behavior without deleting the injected classes. 
If you don't need the columns to be resizable, you can try to set: `resizableColumns: false`

**Reply by chaerror** [reply]
Hi @qwadrox, I understand that it is not good to delete them. The important thing is that you explained to me the reason why they are inserted (resizableColumns: true). I only have the problem on small devices where I styled the table header to my needs. And I don't want to change column width on these devices. So if I set resizableColumns: false on small devices it's actually a solution for me.

Thank you very much for the explanation.
